deeplearning4j-core/src/main/java/org/deeplearning4j/datasets/datavec/RecordReaderDataSetIterator.java

```diff
@@ -29,6 +29,8 @@
 import org.datavec.api.records.reader.RecordReader;
 import org.datavec.api.records.reader.SequenceRecordReader;
 import org.datavec.api.writable.Writable;
+import org.datavec.common.data.NDArrayWritable;
+import org.deeplearning4j.exception.DL4JInvalidInputException;
```

Comment:
And this is your line... and there is a usage of this exception class. And I'm struggling to understand how it could happen. I see that at master branch on github there is no usage.